### ISSUE TITLE
Wrong TenantId example

### ISSUE THREAD

**Issue by StefanSchoof** [MicrosoftDocs/partner-center-docs-powershell#9]
In Example 3 the example uses <AppId> for the TenantId:

```powershell
Connect-PartnerCenter -Credential $credential -ServicePrincipal -TenantId '<AppId>'
```

This is a little bit confusing, since this must be the TenantId and not the AppId.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 30235979-d3c2-d45e-c691-9b9b32856412
* Version Independent ID: b511e34c-016f-4c53-7449-98ed8416bb4a
* Content: [Connect-PartnerCenter (Partner-Center)](https://docs.microsoft.com/en-us/powershell/module/partnercenter/partner-center/connect-partnercenter?view=partnercenterps-1.5)
* Content Source: [docs/help/Connect-PartnerCenter.md](https://github.com/Microsoft/Partner-Center-PowerShell/blob/master/docs/help/Connect-PartnerCenter.md)
* Service: **partner-center**
* GitHub Login: @iswillia
* Microsoft Alias: **iswillia**

**Comment by isaiahwilliams** [comment]
Hi @StefanSchoof, than, you bring this to my attention. I will make sure the document gets updated with the appropriate information today.

**Comment by isaiahwilliams** [comment]
Hi @StefanSchoof we just recently merged pull request #10 that included a fix for this issue. Thank you for letting us know about this issue. Please let us know if you find any other items that could be refined.
